types/index.d.ts

```diff
@@ -159,6 +159,10 @@ interface Appeal {
   addendumEvidence?: AdditionalEvidenceDocument[];
   pcqId?: string;
   isDecisionAllowed?: string;
+  updateTribunalDecisionList?: string;
+  typesOfUpdateTribunalDecision?: DynamicList;
```

Comment:
Do we need this field? think we can omit it.

Reply:
removed the case field "typesOfUpdateTribunalDecision" and "decisionAndReasonDocsUpload"